lib/index.js

```diff
@@ -131,7 +131,15 @@ MongoStorage = (function() {
         return callback(null, res);
       })["catch"](callback);
     });
-    return req.pipe(busboy);
+    //return req.pipe(busboy);
+    
+    // There may be a requirement to read the req stream before calling this function. e.g. to do virus scanning in temporary location.
+    // The file stream can again be attached to req.filedata and passed to this function in that case.
```

Comment:
When you store it in temporary location, you store it on the filesystem ? What is the difference between storing it in the filesystem and storing it in database ?